pom.xml

```diff
@@ -312,21 +314,271 @@
+                        <groupId>org.apache.hive</groupId>
+                        <artifactId>hive-common</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.apache.hive</groupId>
+                        <artifactId>hive-metastore</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.apache.hive</groupId>
+                        <artifactId>hive-serde</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.apache.hive</groupId>
+                        <artifactId>hive-service</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.apache.hive</groupId>
+                        <artifactId>hive-service-rpc</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.apache.hive</groupId>
+                        <artifactId>hive-shims</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.apache.curator</groupId>
+                        <artifactId>curator-framework</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.apache.thrift</groupId>
+                        <artifactId>libthrift</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.apache.thrift</groupId>
+                        <artifactId>libfb303</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.apache.zookeeper</groupId>
+                        <artifactId>zookeeper</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.slf4j</groupId>
+                        <artifactId>slf4j-api</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.slf4j</groupId>
+                        <artifactId>slf4j-log4j12</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>log4j</groupId>
+                        <artifactId>log4j</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>commons-logging</groupId>
+                        <artifactId>commons-logging</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.codehaus.groovy</groupId>
+                        <artifactId>groovy-all</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+
+            <dependency>
+                <groupId>org.apache.hive</groupId>
+                <artifactId>hive-serde</artifactId>
```

Comment:
necessary? we don't test hvie serde table if we do we can change those tests to spark default datasource

Reply:
Got a ClassNotFound Exception without this `hive-serde`

Reply:
where do we get this err?

Reply:
In module `kyuubi-main`
```
[INFO] --- scalatest-maven-plugin:2.0.0:test (test) @ kyuubi-main ---
Discovery starting.
10:16:23.436 WARN org.apache.hadoop.util.NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
10:16:28.182 WARN org.apache.directory.server.core.DefaultDirectoryService: You didn't change the admin password of directory service instance 'SparkProcessBuilderSuite'.  Please update the admin password as soon as possible to prevent a possible security breach.
Discovery completed in 5 seconds, 604 milliseconds.
Run starting. Expected test count is: 80
KyuubiOperationPerServerSuite:
10:16:30.947 WARN org.apache.kyuubi.operation.KyuubiOperationPerServerSuite:

= THREAD AUDIT POST ACTION CALLED WITHOUT PRE ACTION IN SUITE o.a.k.operation.KyuubiOperationPerServerSuite ==

*** RUN ABORTED ***
  java.lang.NoClassDefFoundError: org/apache/hadoop/hive/serde2/thrift/Type
  at org.apache.hive.service.cli.TypeDescriptor.<init>(TypeDescriptor.java:47)
  at org.apache.hive.service.cli.ColumnDescriptor.<init>(ColumnDescriptor.java:46)
  at org.apache.hive.service.cli.TableSchema.<init>(TableSchema.java:46)
  at org.apache.hive.jdbc.HiveQueryResultSet.retrieveSchema(HiveQueryResultSet.java:264)
  at org.apache.hive.jdbc.HiveQueryResultSet.<init>(HiveQueryResultSet.java:198)
  at org.apache.hive.jdbc.HiveQueryResultSet$Builder.build(HiveQueryResultSet.java:179)
  at org.apache.hive.jdbc.HiveDatabaseMetaData.getCatalogs(HiveDatabaseMetaData.java:148)
  at org.apache.kyuubi.operation.BasicJDBCTests.$anonfun$$init$$2(BasicJDBCTests.scala:27)
  at org.apache.kyuubi.operation.BasicJDBCTests.$anonfun$$init$$2$adapted(BasicJDBCTests.scala:25)
  at org.apache.kyuubi.operation.JDBCTestUtils.$anonfun$withMultipleConnectionJdbcStatement$3(JDBCTestUtils.scala:42)
  ...
  Cause: java.lang.ClassNotFoundException: org.apache.hadoop.hive.serde2.thrift.Type
  at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
  at java.lang.ClassLoader.loadClass(ClassLoader.java:419)
  at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
  at java.lang.ClassLoader.loadClass(ClassLoader.java:352)
  at org.apache.hive.service.cli.TypeDescriptor.<init>(TypeDescriptor.java:47)
  at org.apache.hive.service.cli.ColumnDescriptor.<init>(ColumnDescriptor.java:46)
  at org.apache.hive.service.cli.TableSchema.<init>(TableSchema.java:46)
  at org.apache.hive.jdbc.HiveQueryResultSet.retrieveSchema(HiveQueryResultSet.java:264)
  at org.apache.hive.jdbc.HiveQueryResultSet.<init>(HiveQueryResultSet.java:198)
  at org.apache.hive.jdbc.HiveQueryResultSet$Builder.build(HiveQueryResultSet.java:179)
```

Reply:
all right.